src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -400,8 +400,10 @@ protected void printLibraryVersions() {
                 final Attributes manifestAttributes = manifest.getMainAttributes();
                 final String htsjdkVersion = manifestAttributes.getValue("htsjdk-Version");
                 final String picardVersion = manifestAttributes.getValue("Picard-Version");
+                final String sparkVersion = manifestAttributes.getValue("Spark-Version");
                 logger.info("HTSJDK Version: " + (htsjdkVersion != null ? htsjdkVersion : "unknown"));
                 logger.info("Picard Version: " + (picardVersion != null ? picardVersion : "unknown"));
+                logger.info("Spark Version: " + (sparkVersion != null ? sparkVersion : "unknown"));
```

Comment:
I would say something like "Built for spark version:" because it's very possible that they are trying to run it against a different version.

Reply:
It would be good if we could actually figure out what the actual given spark version is as well, but it might be tricky given that the actual spark installation could be on a different machine.  I think it gets logged already too.

Reply:
Good point -- the idea is to document for the user what the officially supported/recommended version of Spark is. I'll adjust the language.